modules/nw-cudn-best-practices.adoc

```diff
@@ -0,0 +1,27 @@
+//module included in the following assembly:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="considerations-for-cudn_{context}"]
+= Best practices for ClusterUserDefinedNetwork CRs
+
+Before setting up a `ClusterUserDefinedNetwork` custom resource (CR), users should consider the following information:
+
+* A `ClusterUserDefinedNetwork` CR is intended for use by cluster administrators and should not be used by non-administrators. If used incorrectly, it might result in security issues with your deployment, cause disruptions, or break the cluster network. 
+
+* `ClusterUserDefinedNetwork` CRs should not be created in the default namespace.
+
+* `ClusterUserDefinedNetwork` CRs should not target `openshift-*` namespaces.
+
+* {product-title} administrators should be aware that empty an `matchLabel` selector results in selecting all namespaces of the cluster.
+
+* For primary networks, the namespace used for the `ClusterUserDefinedNetwork` CR must include the `k8s.ovn.org/primary-user-defined-network` label. This label cannot be updated, and can only be added when the namespace is created. The following conditions apply with the `k8s.ovn.org/primary-user-defined-network` namespace label:
```

Comment:
Please note this is true for primary UserDefinedNetwork CRs as well.
It should probably addressed in follow-up PR if we dont have it already.

Reply:
Yep--it's documented in this PR: https://github.com/openshift/openshift-docs/pull/86845/files#diff-a4571cfec54163a4dbb46db34db71f3ea359a229f8c22eb0413727622160a934R20